src/components/Footer/social-icons/index.ts

```diff
@@ -0,0 +1,2 @@
+export { SocialIcon } from "./social-icon.tsx";
```

Comment:
do you need this export? from what I see it used only in SocialIcons component
in this case better to rename social-icons.tsx => index.tsx, and delete this file